qa/tasks/cephfs/filesystem.py

```diff
@@ -713,6 +713,11 @@ def create(self):
     def run_client_payload(self, cmd):
         # avoid circular dep by importing here:
         from tasks.cephfs.fuse_mount import FuseMount
+
+        # Wait for mds.0 to reach up::active state before mounting
+        # cephfs client in run_client_payload()
+        self.wait_for_state('up:active', timeout=30, rank=0)
+
```

Comment:
a timeout would still mount?

Reply:
If after `30` seconds the rank 0 still couldn't reach `up:active` state, the `wait_for_state()` will `raise RuntimeError`.